Features/CopyMoveDelete/CopyContent.feature

```diff
@@ -0,0 +1,31 @@
+Feature: Copy content
+    In order to copy objects
+    As an Editor user
+    I need to be able to copy an object that I am viewing
+
+    Background:
+        Given I am logged in as an Editor in PlatformUI
+
+    ##
+    #Copy objects to valid locations
+    ##
+    @javascript
+    Scenario: Copy one object without children objects
+        Given a "Destiny" folder exists
```

Comment:
I gotta ask... "destiny" ? Is it like the Fate folder ? :-)

Jokes aside, did you mean "Destination" ?